src/commtrack.js

```diff
@@ -402,9 +426,6 @@ define([
         },
         loadXML: function () {
             this.__callOld();
-            this.data.core.form.on("question-remove", function (event) {
-                dropSetValues(event.mug);
-            });
         }
```

Comment:
Looks like you can remove this entire method since it's only doing a super call now.